model_tests.rb

```diff
@@ -689,14 +689,22 @@ def test_pv_and_storage_demandleveling_osm
     result = sim_test('pv_and_storage_demandleveling.osm')
   end
 
+  def test_refrigeration_system_rb
+    result = sim_test('refrigeration_system.rb')
+  end
+
   def test_refrigeration_system_osm
     result = sim_test('refrigeration_system.osm')
   end
 
-  def test_refrigeration_system_rb
-    result = sim_test('refrigeration_system.rb')
+  def test_refrigeration_system_2_rb
+    result = sim_test('refrigeration_system_2.rb')
   end
 
+  # def test_refrigeration_system__2osm
```

Comment:
This is missing the comment `# TODO: To be added in the next official release after: 3.1.0`, high level tests are failing as a result.

Reply:
Done in 8de6496e463e80bf47a84fef1d03e0ccd1a3213b